cluster/src/assembly/resources/conf/iotdb-cluster.properties

```diff
@@ -129,7 +129,26 @@ flush_raft_log_threshold=10000
 # The cycle when raft log is periodically forced to be written to disk(in milliseconds)
 # If force_raft_log_period_in_ms = 0 it means force insert raft log to be written to disk after
 # each refreshment. Set this parameter to 0 may slow down the ingestion on slow disk.
-force_raft_log_period_in_ms=10
+force_raft_log_period_in_ms=1000
 
 # Size of log buffer in each RaftMember's LogManager(in byte).
-raft_log_buffer_size=16777216
\ No newline at end of file
+raft_log_buffer_size=16777216
+
+# The maximum value of the raft log index stored in the memory per raft group,
+# These indexes are used to index the location of the log on the disk
+max_raft_log_index_size_in_memory=10000
+
+# The maximum value of the raft log persisted on disk per file(in byte) per raft group
+max_raft_log_persist_data_size_per_file=1073741824
+
+# The maximum number of persistent raft log files on disk per raft group, So each raft group's
+# So each raft group's log takes up disk space approximately equals
+# max_raft_log_persist_data_size_per_file*max_number_of_persist_raft_log_files
+max_number_of_persist_raft_log_files=5
+
+# The maximum number of logs saved on the disk
+max_persist_raft_log_number_on_disk=1000000
```

Comment:
I doubt if this parameter is necessary, given that you have already prevented storage abuse by taking two parameters: `max_number_of_persist_raft_log_files` and `max_raft_log_persist_data_size_per_file`